src/repl.zig

```diff
@@ -23,12 +23,13 @@ pub const Parser = struct {
     terminal: *const Terminal,
 
     pub const Error = error{
-        BadKeyValuePair,
-        BadValue,
-        BadOperation,
-        BadIdentifier,
-        MissingEqualBetweenKeyValuePair,
-        NoSyntaxMatch,
+        KeyValuePairBad,
+        ValueBad,
+        OperationBad,
+        IdentifierBad,
+        KeyValuePairEqualMissing,
+        SyntaxMatchNone,
+        SliceOperationUnsupported,
```

Comment:
Just for better aligning, move `KeyValuePairEqualMissing` up, close to `KeyValuePairBad`.

**Short note on naming things (it's hard!):**

The TigerStyle tells us to put the most significant word first, so things such as "the maximum size of a batch" should be named `batch_size_max` instead of `max_batch_size`.

However, it's at our discretion to identify the most significant word for the context.
One great example, the results `exceeds_credits` and `exceeds_debits`, where "exceeds" is in fact the reason why the transfer failed. 

One could argue why "Bad" isn't the most important word in this particular case, to align `BadKeyValuePair`, `BadValue`, `BadOperation`: Since it's an enum for parser error, we expect that all values would be something related to "bad", "invalid", "missing" or whatever ... so those words add very little meaning in the name.

Reply:
Makes sense! They are already aligned elsewhere in the code (ex: https://github.com/tigerbeetle/tigerbeetle/blob/main/src/repl.zig#L903) so I will align it with those two occurrences.